recipe/patches/002-find-protobuf.patch

```diff
@@ -0,0 +1,102 @@
+diff --git a/src/frontends/onnx/onnx_common/CMakeLists.txt b/src/frontends/onnx/onnx_common/CMakeLists.txt
```

Comment:
I supposed nothing has changed in terms of protobuf between 2023.0.1 and 2023.0.2, is it?

Reply:
For some reason it can't find protobuf
```
CMake Error at thirdparty/CMakeLists.txt:434 (find_package):
  Could not find a package configuration file provided by "Protobuf"
  (requested version 3.20.3) with any of the following names:

    ProtobufConfig.cmake
    protobuf-config.cmake
    protobufConfig.cmake
    protobuf-config.cmake

  Add the installation prefix of "Protobuf" to CMAKE_PREFIX_PATH or set
  "Protobuf_DIR" to a directory containing one of the above files.  If
  "Protobuf" provides a separate development package or SDK, be sure it has
  been installed.
```
but protobuf was installed
```
2023-08-30T12:25:01.6577434Z     libprotobuf:        3.20.3-h3eb15da_0   conda-forge
2023-08-30T12:25:01.6577878Z     libprotobuf-static: 3.20.3-h3eb15da_0   conda-forge
```

Here is [link to raw logs](https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/771730/logs/25) from first run. Affected all platforms

Reply:
Let's try to install `libprotobuf-python-headers` as well (see https://github.com/conda-forge/libprotobuf-feedstock/commit/53a636d18ec6602eef85c5a0174cbbe7941ecb27), but not sure whether it helps